otherlibs/site/src/plugins/linker/dune

```diff
@@ -0,0 +1,5 @@
+(library
+ (name dune_site_backend)
+ (public_name dune-site.linker)
+ (virtual_modules linker)
```

Comment:
Could you leave a comment here to explain why this is virtual?

Reply:
> Could you leave a comment here to explain why this is virtual?

Will do.